src/forms/Textarea.jsx

```diff
@@ -138,35 +139,32 @@ class Textarea extends React.Component {
 		}
 
 		return (
-			<div>
-				<div className="inputContainer">
-					{label &&
-						<label className={classNames.label} htmlFor={id}>
-							{label}
-						</label>
-					}
-					{helperText &&
-						<div className={classNames.helperText}>
-							{helperText}
-						</div>
-					}
-					<textarea
-						type='text'
-						name={name}
-						required={required}
-						className={classNames.textarea}
-						onChange={this.onChange}
-						rows={rows}
-						ref={(textarea) => {this.textarea = textarea;}}
-						style={{ ...style, ...heightConstraints }}
-						id={id}
-						value={this.state.value}
-						{...other}
-					/>
-
-					{ maxLength && <p tabIndex="-1" className='text--tiny text--secondary align--right charCount'>{parseInt(maxLength - this.state.value.length)}</p> }
-				</div>
-				{ error && <p className='text--error text--small'>{error}</p> }
+			<div className="inputContainer">
+				{label &&
+					<label className={classNames.label} htmlFor={id}>
+						{label}
+					</label>
+				}
+				{helperText &&
+					<div className={classNames.helperText}>
+						{helperText}
+					</div>
+				}
+				<textarea
+					type='text'
+					name={name}
+					required={required}
+					className={classNames.textarea}
+					onChange={this.onChange}
+					rows={rows}
+					ref={(textarea) => {this.textarea = textarea;}}
+					style={{ ...style, ...heightConstraints }}
```

Comment:
nitpick, but you could just set `style.min/maxHeight` explicitly instead of creating a new `heightConstraints` object and then spreading `style` and `heightConstraints` here